tools/checkstyle/checkstyle.bzl

```diff
@@ -0,0 +1,191 @@
+def checkstyle_dependencies(
+    omit = [],
+    versions = {
+      "antlr_antlr": "2.7.7",
+      "org_antlr_antlr4_runtime": "4.7.1",
+      "com_puppycrawl_tools_checkstyle": "8.15",
+      "commons_beanutils_commons_beanutils": "1.9.3",
+      "info_picocli_picocli": "3.8.2",
+      "commons_collections_commons_collections": "3.2.2",
+      "com_google_guava_guava23": "23.0",
+      "org_slf4j_slf4j_api": "1.7.7",
+      "org_slf4j_slf4j_jcl": "1.7.7",
+    }
+):
+  if not "antlr_antlr" in omit:
```

Comment:
Some of these we import separately using `bazel_deps` right?
Is there a way to use `bazel_deps` imports instead?

Reply:
@oferb I'll check what we can `omit` from there 🙂